scripts/start_local.sh

```diff
@@ -24,11 +25,22 @@ if [ -z "$ARGO_CLOUDOPS_GIT_AUTH_METHOD" ]; then
     export ARGO_CLOUDOPS_GIT_AUTH_METHOD=https
 fi
 
-# TODO / HACK: Vault was not loading credentials from the default chain.
+# Vault was not loading credentials from the default chain, try to fetch from profile
 if [ -n "${AWS_PROFILE}" ]; then
-    export AWS_ACCESS_KEY_ID=`aws configure get $AWS_PROFILE.aws_access_key_id`
-    export AWS_SECRET_ACCESS_KEY=`aws configure get $AWS_PROFILE.aws_secret_access_key`
-    export AWS_SESSION_TOKEN=`aws configure get $AWS_PROFILE.aws_session_token`
+    CREDS_PROCESS_VALUE=`aws configure get $AWS_PROFILE.credential_process`
+    if [ -n "$CREDS_PROCESS_VALUE" ]; then
+        # profile is using credential_process - parse json from it
+        CREDS_JSON=`$CREDS_PROCESS_VALUE`
+        export AWS_ACCESS_KEY_ID=`echo "$CREDS_JSON" | jq -r ."AccessKeyId"`
+        export AWS_SECRET_ACCESS_KEY=`echo "$CREDS_JSON" | jq -r ."SecretAccessKey"`
+        export AWS_SESSION_TOKEN=`echo "$CREDS_JSON" | jq -r ."SessionToken"`
+    else
+        # profile isn't using credential_process, get values from profile config
+        export AWS_ACCESS_KEY_ID=`aws configure get $AWS_PROFILE.aws_access_key_id`
+        export AWS_SECRET_ACCESS_KEY=`aws configure get $AWS_PROFILE.aws_secret_access_key`
+        export AWS_SESSION_TOKEN=`aws configure get $AWS_PROFILE.aws_session_token`
+        export ACCOUNT_ID=`aws sts get-caller-identity --query Account --output text`
```

Comment:
This seems redundant with the call below?

Reply:
yes my bad. removing